### PR TITLE
Feat/hng 12 super admin nav bar component

### LINKED ISSUE #12: [FEAT] Super Admin Nav Bar Component
### Description

Implement the Super Admin Nav Bar using ShadCN UI components. The Nav Bar should adhere strictly to the design specifications provided. Below are the detailed instructions and requirements.

**Requirements:**

1. **UI Components:**
   - Use ShadCN UI components for consistency.
   - Ensure all components match the design specifications provided.

2. **Design Specifications:**
   - Refer to the provided design for dimensions, colors, and layout.
   - Include all items as per the design: search bar, profile dropdown, notifications and help.

3. **Functionality:**
   - Include a dropdown for the user profile with options such as "Settings," "Profile," and "Logout."
   - Implement a help icon with a drop down showing options like "customer care", "FAQ"
   - Implement a notifications icon with a badge showing the number of unread notifications.

4. **Accessibility:**
   - Ensure all components are accessible and follow best practices

5. **Testing:**
   - Ensure the component renders correctly with all required elements.

**Acceptance Criteria:**

- [ ] The `SuperAdminNavBar` component is created and located in the `components` directory.
- [ ] All UI components are implemented using ShadCN UI.
- [ ] The Nav Bar matches the design specifications.
- [ ] The Nav Bar functions correctly 
- [ ] User profile dropdown is functional with appropriate options.
- [ ] Notifications icon with badge is implemented.
- [ ] All components are accessible 

**Attachments:**
Screenshot 2024-07-17 at 13 07 35
[Link to Design file](https://www.figma.com/design/VEItfX6St5NSAqqNHImcxD/HNG-Boilerplate-Designs?node-id=2179-37958&m=dev)

- [ShadCN UI Documentation](https://ui.shadcn.com/)

### PR DESCRIPTION
**Pull request by taiwonaf**
# Description

**Closes #12**

# Changes proposed

## What were you told to do?
- Created Super admin nav bar component
- Created admin layout component
- created admin route (/admin)

## https://github.com/hngprojects/hng_boilerplate_nextjs/issues/12

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title and description of the PR is clear and explains the approach.
- [X] I am making a pull request against the **dev branch** (left side).
- [X] My commit messages styles matches our requested structure.
- [X] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/Videos
localhost_3000_admin

### CODE REVIEW

File: tailwind.config.ts [hunk 1]
```diff
@@ -52,6 +52,10 @@ const config = {
           DEFAULT: "hsl(var(--card))",
           foreground: "hsl(var(--card-foreground))",
         },
+        error: "#F81404",
```
Comment (michaelhpet) [inline tailwind.config.ts]: Please use variables

**Review by michaelhpet — CHANGES_REQUESTED** [review]

File: tailwind.config.ts (hunk 1 above)
Comment (taiwonaf) [reply tailwind.config.ts]: done

File: src/app/globals.css [hunk 2]
```diff
@@ -33,6 +33,9 @@
     --ring: 222.2 84% 4.9%;
 
     --radius: 0.5rem;
+    --error: #F81404;
```
Comment (michaelhpet) [inline src/app/globals.css]: use hsl values like the variables above

**Review by michaelhpet — CHANGES_REQUESTED** [review]

File: src/app/globals.css (hunk 2 above)
Comment (taiwonaf) [reply src/app/globals.css]: resolved

**Review by michaelhpet — APPROVED** [review]

**Pull request merged**